scancode/scanCode.py

```diff
@@ -224,6 +238,19 @@ def read_scan_options(config):
         raise Exception(ERR_REQUIRED_SECTION % SECTION_OPTIONS)
 
 
+def read_regex(config):
+    """Read the Regular Expressions from the configuration file."""
+    options_dict = get_config_section_dict(config, SECTION_REGEX)
+    # vprint("options_dict: " + str(options_dict))
```

Comment:
@houshengbo we are leaving in vprintf in case we want to increase verbose mode in the future, all the "read_" functions have a similar line.